### PR TITLE
Prevent seppuku and bot kill

### PR DESCRIPTION
**Pull request by skeith**
As the title said.

Not tested yet

### CODE REVIEW

**Comment by PaddoInWonderland** [comment]
If you're willing to test it, and it works, I'll merge. ;)

File: kill/kill.py [hunk 1]
```diff
@@ -25,6 +25,12 @@ def __init__(self, bot):
             x = list(self.kills[server.id].keys())
         if x:
             message = self.kills[server.id][random.choice(x)]['text'].format(victim=victim.display_name, killer=author.display_name)
+            if victim.id == author.id:
+                await self.bot.say("Seppuku is not allowed, though, I'd love to see it")
+            elif victim.id == self.bot.user.id:
+                await self.bot.say("Don't even try!")
+            else:
+                await self.bot.say(message)
```
Comment (Aioxas) [inline kill/kill.py]: This would cause a double output. Instead, put message variable from line 27 here

File: kill/kill.py [hunk 2]
```diff
@@ -25,6 +25,12 @@ def __init__(self, bot):
             x = list(self.kills[server.id].keys())
         if x:
             message = self.kills[server.id][random.choice(x)]['text'].format(victim=victim.display_name, killer=author.display_name)
+            if victim.id == author.id:
+                await self.bot.say("Seppuku is not allowed, though, I'd love to see it")
+            elif victim.id == self.bot.user.id:
+                await self.bot.say("Don't even try!")
```
Comment (Aioxas) [inline kill/kill.py]: add a return here as well or set message variable to reflect what the bot would say here

File: kill/kill.py [hunk 3]
```diff
@@ -25,6 +25,12 @@ def __init__(self, bot):
             x = list(self.kills[server.id].keys())
         if x:
             message = self.kills[server.id][random.choice(x)]['text'].format(victim=victim.display_name, killer=author.display_name)
+            if victim.id == author.id:
+                await self.bot.say("Seppuku is not allowed, though, I'd love to see it")
```
Comment (Aioxas) [inline kill/kill.py]: Add a return or set message variable as the message the bot would say.

**Pull request merged**